src/Attestation/Attestation/Commands/RemoveAzureAttestation.cs

```diff
@@ -0,0 +1,125 @@
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System;
+using Microsoft.Rest.Azure;
+using System.Globalization;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Attestation
+{
+    [Cmdlet("Remove", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "Attestation", SupportsShouldProcess = true, DefaultParameterSetName = NameParameterSet)]
+    [OutputType(typeof(bool))]
+    public class RemoveAzureAttestation : AttestationManagementCmdletBase
+    {
+        #region Parameter Set Names
+
+        private const string NameParameterSet = "ByAvailableAttestation"; 
+        private const string ResourceIdParameterSet = "ResourceIdByAvailableAttestation";
+        private const string InputObjectParameterSet = "InputObjectByAvailableAttestation";
+
+
+        #endregion
+
+        #region Input Parameter Definitions
+        /// <summary>
+        /// Attestation name
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 0,
+            ParameterSetName = NameParameterSet,
+            HelpMessage = "Specifies the name of the attestation to remove.")]
+        [ResourceNameCompleter("Microsoft.Attestation", "ResourceGroupName")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+
+        /// <summary>
+        /// Attestation object
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 0,
+            ParameterSetName = ResourceIdParameterSet,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Attestation Resource Id.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+
+        /// <summary>
+        /// Attestation object
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 0,
+            ParameterSetName = InputObjectParameterSet,
+            ValueFromPipeline = true,
+            HelpMessage = "Attestation object to be deleted.")]
+        [ValidateNotNullOrEmpty]
+        public PSAttestation InputObject { get; set; }
+
+
+        /// <summary>
+        /// Resource group to which the attestation belongs.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 1,
+            ParameterSetName = NameParameterSet,
+            HelpMessage = "Specifies the name of resource group for Azure attestation to remove.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty()]
+        public string ResourceGroupName { get; set; }
+
+
+        /// <summary>
+        /// If present, do not ask for confirmation
+        /// </summary>
+        [Parameter(Mandatory = false,
+        HelpMessage = "Indicates that the cmdlet does not prompt you for confirmation. By default, this cmdlet prompts you to confirm that you want to delete the attestation.")]
+        public SwitchParameter Force { get; set; }
```

Comment:
@shleiAmy the `-Force` parameter should only be used in cases where we want to have an additional prompt to the user for confirmation (_e.g._, if you're deleting a parent resource, you may want to prompt that the user that all children resources under this parent will also be deleted). I don't believe that's the case here, but please correct me if I'm wrong.

Reply:
Removed the -Force parameter